pkg/api-server/definitions/service_insight.go

```diff
@@ -0,0 +1,17 @@
+package definitions
+
+import (
+	"github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+)
+
+var ServiceInsightWsDefinition = ResourceWsDefinition{
+	Name: "Service Insight",
+	Path: "service-insights",
```

Comment:
the pathname should be `service+insights` to be consistent with Dataplane Insights